app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -607,6 +607,7 @@ protected void deliverError(CommCareSetupActivity receiver, Exception e) {
                                     }
                                 }
                             };
+                    mTask.connect(this);

```

Comment:
If you change `RetrieveParseVerifyMessageTask mTask` to `RetrieveParseVerifyMessageTask<CommCareSetupActivity> mTask` it gets rid of the lint warning here